common/src/main/java/com/microsoft/identity/common/internal/telemetry/AndroidTelemetryContext.java

```diff
@@ -60,9 +61,15 @@ private void addApplicationInfo(@NonNull final Context context) {
             final PackageInfo packageInfo = packageManager.getPackageInfo(context.getPackageName(), 0);
             long versionCode = PackageInfoCompat.getLongVersionCode(packageInfo);
 
-            super.addApplicationInfo(packageInfo.applicationInfo.packageName,
+            final ApplicationInfo applicationInfo = context.getApplicationInfo();
+            final String applicationName = applicationInfo.labelRes == 0 ? applicationInfo.nonLocalizedLabel.toString() : context.getString(applicationInfo.labelRes);
```

Comment:
Can you add some comments here? 

I believe for our purposes this is the name of the broker hosting application?